app/src/components/app.jsx

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 import css from '../../styles/index.scss';
 
-const ACCESS_TOKEN_REGEX = /#access_token=([a-zA-Z0-9.\-\_]*)(&[a-z=])?/g;
+const ACCESS_TOKEN_REGEX = /#access_token=([a-zA-Z0-9.\-_]*)(&[a-z=])?/g;
```

Comment:
I dont feel like it's (un)safe, I just had bad past experiences in manipulating working regexes. In any case, it seems to be working on my end, so let's leave the change in